### DISCUSSION TITLE
Xunit.Sdk.MultipleException Empty

### DISCUSSION THREAD

**Discussion by mynameisjaycie** [xunit/xunit#2517] [Question] [answered]
Hi,

I'm somewhat new to using assertions, and I just came across `Assert.Multiple` which shipped in [xunit.assert 2.4.2-pre.12 ](https://www.nuget.org/packages/xunit.assert/2.4.2-pre.12).

I'm attempting to use this new method, but am encountering an issue.

Consider this sample BUnit test, which runs on XUnit:
```c#
[Fact]
public void AssertMultiples()
{
  // Setup mocks

  // Register services

  IRenderedComponent<MyComponent> cut = RenderComponent<MyComponent>();

  IReadOnlyList <IRenderedComponent<MyChildComponent>> MyChildComponent = cut.FindComponents<MyChildComponent>();

  bool myBool = MyChildComponent[0].Instance.MyBool;
  string myString = MyChildComponent[0].Instance.MyString;

  Assert.Multiple(
	() => Assert.True(myBool),
	() => Assert.True(myString == "Hello World"!));
}
```

When I run the test, it results in the following:
```c#
Xunit.Sdk.MultipleException
Multiple failures were encountered:
Inner stack traces:
```

Upon reviewing the [source code](https://github.com/xunit/assert.xunit/blob/68565e58d7e0fc52037e022ea7852bf01854b5fe/MultipleAsserts.cs), it appears it should report every exception encountered, so I'm wondering if I'm setting up my test incorrectly?

Thanks in advance!

**Comment by bradwilson** [comment, marked as answer]
Looks like a bug! Filed: #2556
